Flow.Launcher.Plugin.PythonTemplate/plugin/ui.py

```diff
@@ -28,7 +34,7 @@ def sendActionMess(self, title: str, subtitle: str, method: str, value: List):
         action = copy.deepcopy(ACTION_TEMPLATE)
         action['JsonRPCAction']['method'] = method
         action['JsonRPCAction']['parameters'] = value
-        message.update(action)
+        message['JsonRPCAction'] = action;
```

Comment:
Is this your meaning?

```python
# information
message = copy.deepcopy(RESULT_TEMPLATE)
message['Title'] = title
message['SubTitle'] = subtitle

# action
action = copy.deepcopy(ACTION_TEMPLATE)
action['method'] = method
action['parameters'] = value

message.update(action)
```

Reply:
yes, but I would like to change Action-Template without the wrapper, because Flow's JsonRPCRequestModel is without the wrapper. This aligns with your design that RESULT_TEMPLATE is not `{result:{~~~}}`.